dvc/command/plots.py

```diff
@@ -203,16 +209,20 @@ def _add_props_arguments(parser):
         "--title", default=None, metavar="<text>", help="Plot title."
     )
     parser.add_argument(
-        "--x-label", default=None, metavar="<text>", help="X axis label."
+        "--x-label", default=None, help="X axis label.", metavar="<text>"
     )
     parser.add_argument(
-        "--y-label", default=None, metavar="<text>", help="Y axis lebel."
+        "--y-label", default=None, help="Y axis label.", metavar="<text>"
     )
 
 
 def _add_output_arguments(parser):
     parser.add_argument(
-        "-o", "--out", default=None, help="Destination path to save plots to.",
+        "-o",
+        "--out",
+        default=None,
+        help="Destination path to save plots to.",
```

Comment:
```suggestion
        help="Destination path to save plots to",
```

Reply:
@jorgeorpinel Is it really worth bothering with dots in this PR? We have dots all over the place, I don't really see a point in blocking this PR just because of it.